README.md

```diff
@@ -1,5 +1,105 @@
-This repo cntains all the resource files for setting up Kata with SRIOV device plugin.
+# kata-sriov-dp
 
-It contains a daemonset to copy binaries for multus,sriov,cni-shim and patched kubelet
-to the host and start the sriov device plugin.
-Also included are multus CRD and a test pod yaml for Kata.
+This repository contains all the resource files for setting up [SRIOV](https://github.com/intel/sriov-network-device-plugin) device plugin on a Kubernetes cluster.
+
+`kata-sriov-dp` provides a Dockerfile that builds [Multus CNI](https://github.com/intel/multus-cni) plugin, [SRIOV CNI](https://github.com/intel-corp/sriov-cni) plugin,
+[SRIOV Device Plugin](https://github.com/intel/sriov-network-device-plugin) and a patched version of kubelet 1.10 that is required for the device plugin.
+A deamonset is also provided that runs the SRIOV device plugin itself.
+
+The steps to set up the device plugin assume you have a Kubernetes cluster setup, with nodes having a SRIOV enabled network card.
+
+## Kata Containers setup
+
+Below instructions to verify the SRIOV Device plugin work with runc.
+To test out Kata with device plugin, setup Kata Containers with Kubernetes. Refer [kata-deploy](https://github.com/egernst/kata-deploy).
+
+## SRIOV Device Plugin Setup
+
+1. Build Docker image.
+
+   ```
+   $ sudo docker build -t sriov-dp
+   ```
+
+   Push the image to Dockerhub.
+   ```
+   $ sudo docker login
+   $ sudo docker push sriov-dp
+   ```
+
+2. Create SRIOV virtual functions for your SRIOV network interface.
+   ```
+   $ echo 64 | sudo tee --append  /sys/bus/pci/devices/0000\:01\:00.0/sriov_numvfs
+   ```
+
```

Comment:
append or overwrite? Also is this more user-friendly?
echo 64 | sudo tee /sys/class/net/<interface_name>/device/sriov_numvfs